migration_guides/70.x.x-to-71.0.0-migration-guide.md

```diff
@@ -61,12 +61,12 @@ This section will list the changes that will impact more than one component
 
         </details>
 
-### **_Update icon sizes naming standard_**
+### **_Update to icon sizes naming standard_**
```

Comment:
I think the wording of this is a bit more clear

### **_Standardisation of breakpoint size naming_**

[**Link to PR**](https://github.com/ONSdigital/design-system/pull/3285)

-   **Description of change:** The breakpoint naming convention has been updated to match other size naming like the padding and margin sizes. The old breakpoint size names like xxs and xxl have been replaced with 2xs, 2xl, and so on, to better align with other size naming.

-   **Reason for change:** The new naming convention standardises how breakpoints are referenced across the design system.

-   **Components impacted:**

    Any component or layout that references breakpoints using the previous naming convention will need to be updated.

-   **Migration steps:**

    1. Locate all instances of breakpoints in your codebase.
    2. Replace old breakpoint names with the new corresponding names: